DigitalLearningSolutions.Web/Attributes/BooleanMustBeTrue.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace DigitalLearningSolutions.Web.Attributes
+{
+    using System.ComponentModel.DataAnnotations;
+
+    public class BooleanMustBeTrue : ValidationAttribute

```

Comment:
This is nice - can you add this attribute to ConfirmDeleteGroupViewModel to replace the Validate function there?

Reply:
It's been added ConfirmDeleteGroupViewModel as requested